### DISCUSSION TITLE
자동예약 버튼이 동작을 안합니다.

### DISCUSSION THREAD

**Discussion by 2MKW** [changdoc/naver-vaccine-macro#257] [Q&A]
개발자님 안녕하세요.
특정 병원의 예약 페이지에 존재하던 자동예약 버튼이 오늘 오전 10시 이후? 부터 안보이는데 저만 그런 것일까요?

**Comment by mimkim** [comment]
엇 저또한 안보입니다. 그래서 글쓰려고 들어왔는데 같은 이슈가 있네. 왜그러는걸까요? 컴퓨터 재부팅했는데도 똑같아요.

**Comment by leesongil** [comment]
설마 막힌걸까요 ㅜㅜ 저도 갑자기 버튼이 안보이네요 ㅜㅜ

**Comment by eclair0305** [comment]
앗 그러네요 ㅜㅜ

**Comment by changdoc** [comment]
음.. 저는 잘 노출이 되고 있는데요. 개발자 도구 여시고 console창에 에러 메시지 같은게 있을까요?

**Reply by 2MKW** [reply]
Error handling response: TypeError: Cannot read property 'innerHTML' of null
    at macro (chrome-extension://fpgafoekdkpmkjjddgdafdpcefdcakja/content.js:163:38)
    at chrome-extension://fpgafoekdkpmkjjddgdafdpcefdcakja/content.js:237:13

이렇게 뜹니다!

**Comment by Lanymph** [comment]
Error handling response: TypeError: Cannot read property 'innerHTML' of null
    at macro (chrome-extension://alfhbmpnlhcpcjjaacapcdnggegicepl/content.js:163:38)
    at chrome-extension://alfhbmpnlhcpcjjaacapcdnggegicepl/content.js:237:13

**Reply by Lanymph** [reply]
저도 에러가 뜨네요 오전까지만 해도 자동 예약 잘 돌아갔는데 지금은 안되네요ㅠㅠ

**Reply by sa2da94** [reply]
ㅈㅓ도 잘되다가 다른곳으로 바꾸니깐 저 에러가 뜨네요... 막힌걸까요

**Reply by changdoc** [reply]
스크립트 동작 중간에 에러가 발생하고 있네요.
예약 신청 페이지는 정상적으로 표시가 된 상태일까요?

**Reply by Lanymph** [reply]
그런가봐여 네이버에서 막았나봐여ㅠㅠㅠ 저는 개발자님덕분에 이미 성공했지만 가족꺼 잡아주려다가ㅠㅠ 막혀서 속상하네용

**Reply by 2MKW** [reply]
네. 제대로 뜨고 있고 버튼만 사라졌어요

**Reply by eclair0305** [reply]
> 스크립트 동작 중간에 에러가 발생하고 있네요.
> 예약 신청 페이지는 정상적으로 표시가 된 상태일까요?

네 페이지는 제대로 나오는데, "자동예약" 버튼 클릭시 버튼이 사라집니다

**Comment by ester-000000** [comment]
저도 10시 기점으로 버튼이 아예 안뜹니다. 다른 병원으로 해도 똑같아요ㅠ

**Comment by changdoc** [comment]
![image](https://user-images.githubusercontent.com/1879494/129128064-a23e200d-82f3-4876-b58e-460f17848ebb.png)
아하 대상 병원에 이렇게 나와있으면 오류가 발생하네요 바로 수정 진행해서 업로드 해두겠습니다.

**Comment by changdoc** [comment]
우선 아스트라제네카가 열려있지 않는 병원에서 동작 시키고 계셔야할것 같습니다~

**Reply by leesongil** [reply]
네 개발자님 아스트라제네카 안열리는곳도 현재 안되네요 ㅜㅜ 한번 에러 나면 안되는건가요?ㅜㅜ

**Reply by changdoc** [reply]
아스트라제네카 안열린곳에도 동일한 에러가 나나요?

**Reply by leesongil** [reply]
네 맞습니다 ㅜㅜ

**Reply by ahsneokwon** [reply]
저도 어제는 잘됬는데 오늘다시해보니 도사님이 올려주신 백신종류 선택 이없어도 자동새로고침이 안됩니다.

**Reply by changdoc** [reply]
에러 날 때 예약 신청 화면 스크린샷 공유 가능하실까요? 저는 재현이 안되서요

**Reply by ahsneokwon** [reply]
![제목 없음](https://user-images.githubusercontent.com/88811118/129128818-5f30781a-7320-4d81-b4aa-cabaacc74a39.png)
이렇게 맨처음 저기 검정박스에 자동예약버튼 클릭하면 어제는 자동으로 새로고침 되었는데 오늘은 그버튼이 없어지고 저상태로 가만히 있습니다.

**Reply by ahsneokwon** [reply]
![제목 없음2](https://user-images.githubusercontent.com/88811118/129129250-5eb1f99d-4f7f-403f-a64c-b4b09a71ff79.png)
Error handling response: TypeError: Cannot read property 'innerHTML' of null
    at macro (chrome-extension://mflolkaaijailoicmobkpkmjfndlhnig/content.js:163:38)
    at chrome-extension://mflolkaaijailoicmobkpkmjfndlhnig/content.js:237:13

**Reply by Lanymph** [reply]
> 이렇게 맨처음 저기 검정박스에 자동예약버튼 클릭하면 어제는 자동으로 새로고침 되었는데 오늘은 그버튼이 없어지고 저상태로 가만히 있습니다.

와 저도 이래용 ㅠㅠ

**Comment by changdoc** [comment]
https://github.com/changdoc/naver-vaccine-macro/releases/tag/v1.1.6
에러 나는 부분 수정 버전 으로 시도해봐주세요.

**Reply by ahsneokwon** [reply]
잘되요~ 감사합니다

**Reply by leesongil** [reply]
감사합니다^^ 최고입니다.

**Reply by Lanymph** [reply]
왕 감사합니당♥ 너무 좋아용ㅎㅎ

**Reply by sdr03095** [reply]
ㅠㅠ혹시 이거 어떻게 사용하는 건가ㅏ요?

**Reply by changdoc** [reply]
https://github.com/changdoc/naver-vaccine-macro/issues/29

**Reply by mimkim** [reply]
다운받았는데 ㅠㅠ 저만 안되는건가요 아직 제가 사용법을 잘못하고있는건가요 ㅠㅠ

**Reply by mimkim** [reply]
잘됩니다 감사합니다! :)
